src/registrar/models/domain.py

```diff
@@ -1012,33 +1188,66 @@ def _fetch_cache(self, fetch_hosts=False, fetch_contacts=False):
                 # no point in removing
                 cleaned["hosts"] = []
                 for name in cleaned["_hosts"]:
-                    # we do not use _get_or_create_* because we expect the object we
-                    # just asked the registry for still exists --
-                    # if not, that's a problem
-                    req = commands.InfoHost(name=name)
-                    data = registry.send(req, cleaned=True).res_data[0]
-                    # extract properties from response
-                    # (Ellipsis is used to mean "null")
-                    host = {
-                        "name": name,
-                        "addrs": getattr(data, "addrs", ...),
-                        "cr_date": getattr(data, "cr_date", ...),
-                        "statuses": getattr(data, "statuses", ...),
-                        "tr_date": getattr(data, "tr_date", ...),
-                        "up_date": getattr(data, "up_date", ...),
-                    }
-                    cleaned["hosts"].append(
-                        {k: v for k, v in host.items() if v is not ...}
-                    )
-
+                    # For reviewers - slight refactor here
+                    # as we may need this for future hosts tickets
+                    # (potential host mapper?).
+                    # Can remove if unnecessary
+                    cleaned["hosts"].append(self._get_host_as_dict(name))
```

Comment:
why was hosts updated here? Did a bug occur? Just asking as this will affect @therealslimhsiehdy's ticket and may need to be factored in.

Reply:
I think I'll remove this particular change, I was initially thinking this could've been impacted by the EPPLib stuff (like contacts where you have to deal with weird objects) so I was just getting some stuff moved over for that, but that didn't turn out to be an issue. Code leftover pretty much